tools/hive/hadoop-hive/scripts/hadoop-bootstrap.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
+
+# used by Hadoop
+/usr/sbin/sshd
+
+: ${HADOOP_PREFIX:=/usr/local/hadoop}
+
+$HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+rm /tmp/*.pid
+
+# templating of config files
+sed s/{{HOSTNAME}}/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/core-site.xml.template > /usr/local/hadoop/etc/hadoop/core-site.xml
+sed s/{{HOSTNAME}}/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/yarn-site.xml.template > /usr/local/hadoop/etc/hadoop/yarn-site.xml
+sed s/{{HOSTNAME}}/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/hdfs-site.xml.template > /usr/local/hadoop/etc/hadoop/hdfs-site.xml
+
+# format namenode
+$HADOOP_PREFIX/bin/hdfs namenode -format
+
+# start hdfs and yarn
+$HADOOP_PREFIX/sbin/start-dfs.sh
+$HADOOP_PREFIX/sbin/start-yarn.sh
+
+if [[ $1 == "-bash" ]]; then
```

Comment:
Does this not mean that the container will terminate if the `-bash` argument is not provided?

Reply:
I don't think so. This is a script that gets called from `hive-bootstrap.sh` so it'll just continue on afterward